From 539ba73977ef8afcde317b611b798a21406dac27 Mon Sep 17 00:00:00 2001
From: Nikola Dipanov <ndipanov@redhat.com>
Date: Mon, 10 Jun 2013 12:46:42 +0200
Subject: [PATCH] Remove a runtime dep on python-pbr package

Also patch it out when building docs to have consistent versioning
---
 doc/source/conf.py | 8 ++++++--
 nova/version.py    | 9 +++------
 2 files changed, 9 insertions(+), 8 deletions(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 85e2629..699728f 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -63,11 +63,15 @@ copyright = u'2010-present, OpenStack Foundation'
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-from nova.version import version_info
+from nova.version import version_string
+import pbr.version
+
+version_info = pbr.version.VersionInfo('nova')
+
 # The full version, including alpha/beta/rc tags.
 release = version_info.release_string()
 # The short X.Y version.
-version = version_info.version_string()
+version = version_string
 
 # The language for content autogenerated by Sphinx. Refer to documentation
 # for a list of supported languages.
diff --git a/nova/version.py b/nova/version.py
index f954ef3..7c38a77 100644
--- a/nova/version.py
+++ b/nova/version.py
@@ -14,15 +14,12 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import pbr.version
-
 NOVA_VENDOR = "OpenStack Foundation"
 NOVA_PRODUCT = "OpenStack Nova"
 NOVA_PACKAGE = None  # OS distro package version suffix
 
 loaded = False
-version_info = pbr.version.VersionInfo('nova')
-version_string = version_info.version_string
+version_string = "REDHATNOVAVERSION"
 
 
 def _load_config():
@@ -85,6 +82,6 @@ def package_string():
 
 def version_string_with_package():
     if package_string() is None:
-        return version_info.version_string()
+        return version_string
     else:
-        return "%s-%s" % (version_info.version_string(), package_string())
+        return "%s-%s" % (version_string, package_string())
